### DISCUSSION TITLE
Need help exporting animation to GLB for Spoke

### DISCUSSION THREAD

**Discussion by LuisGuerrero89** [mozilla/hubs#3731] [General Questions & Discussion] [answered]
Hi !

I'm trying to export a simple looping animation of flocking birds, for the birds I'm using an scrollable texture and is working fine, for the animation I just grabbed my plane, offset the center and rotated 360, looks fine in Blender. I know I need to "stash" or "push" the animation to have it as an NLA, however when I do that and set it up on spoke, it doesn't move. I already tried checking and unchecking the "Always Sample Animation"
![image](https://user-images.githubusercontent.com/51933924/105127023-915fa400-5aa5-11eb-9c01-c22bc3139aa8.png)

I've tried also selecting and deselecting the star in the NonLinear Animation panel, renaming everything with the same name to no avail. 
![image](https://user-images.githubusercontent.com/51933924/105126988-7ee56a80-5aa5-11eb-86c2-41bf77548410.png)
![image](https://user-images.githubusercontent.com/51933924/105127060-a805fb00-5aa5-11eb-91a0-00e1c20fd6dc.png)

On a curious note, everything works if I just spawn a new object once I'm inside my hubs scene, but I want the birds to be already in there and with an specific scale. I know I'm missing something but I just can't figure it out on my own, help will be greatly appreciated.

Here is a Zip file with my Blender scene, the sprite sheet and the last .glb [Birds.zip](https://github.com/mozilla/hubs/files/5840176/Birds.zip)

Thanks!!!

**Comment by j-conrad** [comment, marked as answer]
I know this question is a bit old, but I just tried opening your .blend file and just exporting as-is. (I'm using Blender 2.93.0)
The animation exported just fine and I can see it playing in a generic glTF viewer, in Spoke (when previewing), and Hubs.
In Spoke, make sure you select the animation (`Birds`) from the `'Loop Animation'` dropdown menu:
![image](https://user-images.githubusercontent.com/31968496/127236048-3b618309-4dde-4361-b0c1-eda8727bb72c.png)

Hopefully you had already gotten this working, but perhaps there was a temporary issue with the exporter back when you tried it.
